CHANGELOG.rst

```diff
@@ -64,6 +64,11 @@ Breaking changes
  - SumPDF changed in the behavior. Read above in the Major Features and Improvement.
  - Integrals of extended PDFs are not extended anymore.
 
+Depreceations
+-------------
+ - `ComposedParameter` takes now `params` instead of `dependents` as argument, it acts now as
+   the arguments to the `value_fn`.
+
```

Comment:
I think we should state more explicitly that value_fn now needs to take explicit arguments instead of referencing the locals() and globals() dictionaries and implicate that this will force the user to change their value_fn if they want to stay compatible in the future.